app/src/main/kotlin/com/wire/android/feature/conversation/usecase/ResetCurrentConversationIdUseCase.kt

```diff
@@ -0,0 +1,12 @@
+package com.wire.android.feature.conversation.usecase
+
+import com.wire.android.core.exception.Failure
+import com.wire.android.core.functional.Either
+import com.wire.android.core.usecase.UseCase
+import com.wire.android.feature.conversation.data.ConversationRepository
+
+class ResetCurrentConversationIdUseCase(private val conversationRepository: ConversationRepository) :
```

Comment:
Missing tests for this class.

Should at least test that:

* `given the repository succeeds, when reseting the current conversation id, then the success is propagated`
* `given the repository fais, when reseting the current conversation id, then the failure is propagated`
* `given any parameters, when reseting the current conversation id, then repository is used`